tt_eager/tt_dnn/op_library/moreh_sum/moreh_sum_op.cpp

```diff
@@ -49,8 +49,13 @@ namespace {
         const Tensor& input,
         const int64_t& dim,
         const std::optional<const Tensor>& output,
-        const MemoryConfig& output_mem_config) {
-        return operation::run(MorehSum{ .dim = dim, .output_mem_config = output_mem_config }, { input }, {}, { output }).at(0);
+        const MemoryConfig& output_mem_config,
+        std::optional<const DeviceComputeKernelConfig> &compute_kernel_config) {
+
+        TT_FATAL(input.storage_type() == StorageType::DEVICE);
+        auto device = input.device();
+        auto kernel_config_val = init_device_compute_kernel_config(device->arch(), compute_kernel_config);
```

Comment:
```suggestion
        auto kernel_config_val = init_device_compute_kernel_config(device->arch(), compute_kernel_config, MathFidelity::HiFi4);
```
If math fidelity is not set, the default math fidelity is low. Is this ok?

Reply:
Good point. :) I will apply it.